File: csharp/etl/Etl.cs
public static class Etl
{
    public static Dictionary<string, int> Transform(Dictionary<int, string[]> old)
    {
        Dictionary<string, int> newDictionary = new();
        foreach (var scoreLetters in old)
        {
            int score = scoreLetters.Key;
            string[] letters = scoreLetters.Value;
            foreach (string letter in letters)
            {
                newDictionary.Add(letter.ToLower(), score);
            }
        }
        return newDictionary;
    }
}

File: javascript/etl/etl.js
//
// This is only a SKELETON file for the 'ETL' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const transform = (data) => {
  const newData = {};
  for (const [score, letter] of Object.entries(data)){
    letter.forEach(letter => {
      newData[letter.toLowerCase()] = Number(score);
    })
  }
  return newData;
};
